contracts/CourtRoom.sol

```diff
@@ -183,20 +218,41 @@ contract SwearGame is Owned {
 		registeredPlayersCounter--;
 
 	}
-
-
-	function openNewClaim(bytes32 _evidence) public returns (bool) {
+	function openNewClaim(address beneficiary, uint256 blockNumber,
```

Comment:
`openNewClaim` -> `newCase`
and with delegated service validity verification, no need for promise related arguments